### PR TITLE
TC-2206 Build custom fluentd image in Jenkins

### PR DESCRIPTION
**Pull request by keithwoody**
Adds a Jenkinsfile for use in a Jenkins build pipeline. Creates and pushes a Fluentd image to ECR tagged with the **<build_date>-<commit_sha>** and **build<build_num>**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        stage('Build fluentd debian-nfs image') {
+            steps {
+                script {
+                    GIT_COMMIT = sh(returnStdout: true, script: "git log -n 1 --pretty=format:'%h'").trim()
+                    STR_DATE = sh(returnStdout: true, script: "date +\"%y%m%d%H%M\"").trim()
+                    APP_TAG = "${STR_DATE}-${GIT_COMMIT}"
+                }
+
+                sh """
+                    docker build -t $ECR_URL/fluentd -f docker-image/v0.12/debian-nfs/Dockerfile docker-image/v0.12/debian-nfs
```
Comment (maj) [inline Jenkinsfile:39]: It seems like these could be easier to reuse if $ECL_URL/fluentd was a variable

Comment (maj) [reply Jenkinsfile:39]: In other words my guess is most of this is pretty boilerplate between projects

**Review by maj — APPROVED** [review]
Seems logical to me based on my limited understanding of this. Can you confirm that no changes to Jenkins are required for this? In otherwords if you added a file like to this another project would it "just work"?

If so, what's the process to modify the pipeline and get the file back into the repo?

**Review by onewland — APPROVED** [review]

**Comment by keithwoody** [comment]
@maj It's not exactly "drop in a Jenkins file and it works" but the process for adding a pipeline is pretty straight-forward. If a pipeline already exists for a project, then changes to the Jenkinsfile do get picked up automatically.

**Pull request merged**